deploy/README-multicloud.md

```diff
@@ -0,0 +1,100 @@
+# Configuring Service Assurance Framework for Multiple Clouds
+
+Multiple Openstack clouds can be configured to target a single instance of SAF.
+There are a few steps to get this set up:
+
+1. Plan the AMQP address prefixes to use for each cloud
+1. Deploy metrics and notification SmartGateways for each cloud to listen on the
+  corresponding address prefixes
+1. Configure each cloud to send it's metrics and notifications to SAF on the
+  correct address
+
+## AMQP addresses
+
+By default, OSP nodes are configured to send data to the `collectd/telemetry`
+and `collectd/notify` addresses on the AMQP bus; and SAF is configured to
+listen on those addresses for monitoring data. In order to support multiple
+clouds and have the ability to easily identify which cloud genereated which
+monitoring data, each cloud should be configured to send to a unique address.
+
+It is recommended to prefix a cloud identifier to the second part of the
+address. For example:
+
+* collectd/cloud1-telemetry
+* collectd/cloud1-notify
+* collectd/cloud2-telemetry
+* collectd/cloud2-notify
+* collectd/us-east-1-telemetry
+* collectd/us-west-3-telemetry
+* ...etc
+
+## Deploying SmartGateways
+
+Two SmartGateways (one for metrics, one for events) need to be deployed
```

Comment:
Technically you could just get metrics or events, although I'm not sure that pedantry is necessary here :)